src/Microsoft.AspNetCore.Antiforgery/IAntiforgery.cs

```diff
@@ -14,7 +14,7 @@ public interface IAntiforgery
     {
         /// <summary>
         /// Generates an <see cref="AntiforgeryTokenSet"/> for this request and stores the cookie token
-        /// in the response.
+        /// in the response. It also sets the "Cache-control" and "Pragma" headers to "no-cache".
```

Comment:
Replace "It also" with "This operation also". I'd also mention the `X-Frame-Options` as well here if you're going to describe the header actions.